bin/recog_standardize

```diff
@@ -111,7 +111,7 @@ ARGV.each do |arg|
             puts "HW DEVICE MISSING: #{val}"
             hw_device[val] = true
           end          
-        when "service.product"
+        when "service.product", "service.component.product"
```

Comment:
Collect `service.component.product` since the script already does the same for `service.component.vendor`.